status-im/status-react
# Features/pairing prompt

### Description

If a new device is detected, it will prompt the user to pair the devices through a pop up. This pop up will only displayed once for each new `device`, and only if you are not already in the `devices` section.

Currently there are two ways to make a device discoverable:

1) Explicitly, by clicking on `Pair this device`, which reveals information to other devices with the same pk about the `device-type`, eventually the name
2) Implicitly, when pairing is enabled and sending a 1-to-1 message/contact updates. This does not reveal any information to other device with the same pk other than a random id

Also moves `devices` outside `advanced` in desktop to it's own section.

For some of the devices (those implicitly discovered), we don't have info on whether is `desktop` or `mobile`, so probably we should add an icon for this?

 @andmironov @chadyj could you have a quick look if you are ok with ux/copy ?

### Testing

1) Install on device A, create a new account, enable development mode.
2) Install on device B, create a new account, enable development mode.
3) Send a 1-to-1 message to anyone
4) A pop up should be displayed on device B, click on `Go to devices section`
5) You should see a new entry, click on `Enable` and `Pair this device`
6) You should see the same popup on A, go to device section and click on Enable
7) Devices are not paired

status:wip
![pairing](https://user-images.githubusercontent.com/1017008/49376994-3cb41000-f709-11e8-9d19-548384e5e6f8.png)
